### PR TITLE
doc/user: Quickstart real-time application

### PR DESCRIPTION
**Pull request by joacoc**
### Motivation

A new quickstart related to real-time applications and Materialize! 

### Tips for reviewer

It's a general overview of the [Antennas Postgres](https://github.com/MaterializeInc/demos/tree/main/antennas-postgres) demo.

[Preview](https://preview.materialize.com/13026/quickstarts/application/)

### Testing

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.

### Release notes

This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):

  -

### CODE REVIEW

File: doc/user/content/quickstarts/application.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+For a quick start, let’s focus only on the Postgres case, where the source already defines a schema. If you want to know how it would look using Kafka/Redpanda without a schema, refer to [this repository](https://github.com/MaterializeInc/demos/tree/main/antennas-kafka).
+
+The first step while building an application using Postgres is making available the tables to consume from Materialize.
+It’s done through a process known as [Change Data Capture (CDC)](https://materialize.com/docs/sql/create-source/materialize-cdc/). As it refers to in the [integration guide](https://materialize.com/docs/integrations/cdc-postgres/), <i>“it allows you to track and propagate changes in a Postgres database to downstream consumers based on its Write-Ahead Log (WAL).”</i>
+
+From Materialize standpoint it requires creating a source:
+
+```sql
+    CREATE MATERIALIZED SOURCE IF NOT EXISTS antennas_publication_source
```
Comment (sjwiesman) [inline doc/user/content/quickstarts/application.md:66]: The requirement for materializing a postgres source for multiple downstream views goes away in platform. Not sure if we should remove this now or later.

**Review by sjwiesman — COMMENTED** [review]
I started a review and then got pulled away. I'm unsure if I can get back to this until tomorrow, so I'm posting what I have. I like the content and mostly have some phrasing/grammar suggestions to work on the flow. 


Feel free to take or leave most of these suggestions.

File: doc/user/content/quickstarts/application.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+Each antenna has a known fixed range capable of serving customers. Operations teams watch a map of green, yellow, or red circles (antenna’s range area and health status) to understand where the issues are.
+
+A traditional approach like batch processing demands extra time to detect any issue, exposing the clients to a bad experience. A real-time application should be able to keep customers, businesses, and operational people happy.
+
+## Conceptual Overview
+
+### Sources
+
+First, a [source](https://materialize.com/docs/sql/create-source/) needs to spin up. It could be Postgres, Redpanda, Kafka, S3, or other alternatives. It will define how to store, insert, and process in Materialize the performance events and antenna’s position on the map.
```
[the remark this answers is not in the document]
Comment (joacoc) [reply doc/user/content/quickstarts/application.md:43]: It could also be a table or a view, or do you mean another way?

[the remark this answers is not in the document]
Comment (sjwiesman) [reply doc/user/content/quickstarts/application.md:43]: I was just ignoring details

File: doc/user/content/quickstarts/application.md (hunk 1 above)
Comment (joacoc) [reply doc/user/content/quickstarts/application.md:66]: I agree that not having it would be better, but removing the Materialization will end up in a `replication error slot` since it's using two Materialized views downstream.

**Comment by morsapaes** [comment]
Thanks for putting this up, @joacoc! I'll push some suggestions to shuffle the content around a bit, and make sure it follows a consistent structure that can be reused across quickstarts (I have some ideas from reworking the BI one!); as a general guideline, users should be able to start _doing things_ as soon as possible, and get some extra "conceptual" context as they go, instead of in bulk at the beginning of the quickstart. Stashing this for tomorrow, sorry it took me so long to get to it!

File: doc/user/content/quickstarts/application.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+        parent: quickstarts
+weight: 20
+aliases:
+  - /demos/real-time-application
+---
+
+**tl;dr** Materialize unleashes real-time powers for your applications. Process, serve, and display information quickly as a blink.
+
+{{< figure
+    src="https://user-images.githubusercontent.com/11491779/166932582-e5a9fd47-e397-4419-b221-e8f38c6f06f5.mp4"
```
Comment (morsapaes) [inline doc/user/content/quickstarts/application.md:15]: Can you see this rendered? It comes up as a broken link for me locally.

Comment (joacoc) [reply doc/user/content/quickstarts/application.md:15]: It seems that it is not available anymore. Is there any commonplace to store content? I was thinking about using the WordPress storage.

Comment (morsapaes) [reply doc/user/content/quickstarts/application.md:15]: I can access it through the link, just not see it rendering on the docs website. 🤔  I also use the GitHub servers to store content, but Wordpress could be an option. What we could also do is make it a short GIF — there's no real need to have a 16s video for this. WDYT?

Comment (joacoc) [reply doc/user/content/quickstarts/application.md:15]: I would love to! The options I've tried back in time turned it into a super pixelated monster, so I desisted

Comment (morsapaes) [reply doc/user/content/quickstarts/application.md:15]: Giphy usually works pretty well!

Comment (joacoc) [reply doc/user/content/quickstarts/application.md:15]: I just created an account and tried with Giphy https://media.giphy.com/media/plAtuUgaZA90Jr7AzD/giphy.gif
Wdyt?

File: doc/user/content/quickstarts/application.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ aliases:
 **tl;dr** Materialize unleashes real-time powers for your applications. Process, serve, and display information quickly as a blink.
 
 {{< figure
-    src="https://user-images.githubusercontent.com/11491779/166932582-e5a9fd47-e397-4419-b221-e8f38c6f06f5.mp4"
+    src="https://materialize.com/wp-content/uploads/2022/06/Jun-16-2022-18-38-42.gif"
```
Comment (morsapaes) [inline doc/user/content/quickstarts/application.md:15]: 💯

**Comment by joacoc** [comment]
Does not apply anymore.
